code/game/machinery/camera/camera.dm

```diff
@@ -72,9 +67,18 @@ CREATION_TEST_IGNORE_SUBTYPES(/obj/machinery/camera)
 
 /obj/machinery/camera/Initialize(mapload, obj/structure/camera_assembly/CA)
 	. = ..()
-	for(var/i in network)
-		network -= i
-		network += LOWER_TEXT(i)
+	var/static/list/autonames_in_areas = list()
+
+	// Calculate area code
+	var/area/camera_area = get_area(src)
+	if (istype(camera_area, /area/space))
+		var/turf/connected_wall = get_step(src, dir)
+		camera_area = get_area(connected_wall)
+
+	// Calculate the camera tag
+	if (!c_tag)
+		c_tag = "[format_text(camera_area.name)] #[++autonames_in_areas[camera_area]]"
```

Comment:
```suggestion
	if (!c_tag)
		if (istype(camera_area, /area/space))
			c_tag = "[format_text(camera_area.name)] #[++autonames_in_areas[camera_area]] (Space)"
		else
			c_tag = "[format_text(camera_area.name)] #[++autonames_in_areas[camera_area]]"
```
this will be nice if we can notice a camera is located at space or not.

Reply:
This code will only ever append space if the connected wall is space, in which case it will just say "space #1 (space)". What you want is a check to see if the current turf is space but the area is not, which I can add (will append external on in that case)

Reply:
yeah you're right. some cameras that are only intended to show spaces are hard to recognise when the names are the same.